typescript/test/codeActions/addDestruct.spec.ts

```diff
@@ -253,9 +253,80 @@ describe('Add destructure', () => {
             })
         })
     })
+    describe('Should destruct param call expression param', () => {
+        test('Should skip if trying to destruct call expression', () => {
+            const initial = /* ts */ `
+              const /*t*/newVariable/*t*/ = { test: 1}
+
+              const obj = {
+                  tag: foo.map(newVariable.test),
+              }
+            `
+            const expected = /* ts */ `
+              const { test } = { test: 1}
+
+              const obj = {
+                  tag: foo.map(test),
+              }
+            `
+
+            const { codeAction } = fourslashLikeTester(initial, undefined, { dedent: true })
+
+            codeAction(0, {
+                refactorName: 'Add Destruct',
+                newContent: expected,
+            })
+        })
+    })
+    describe.todo('Vue support', () => {
```

Comment:
Is there a way to add tests for vue files? I just need these kind of conditions to return true

https://github.com/zardoy/typescript-vscode-plugins/pull/202/files#diff-27021c5e110bea0d31b7221dfe3ae52a5ab54fff9093bf002b143c31bd96caa3R27

Or maybe just define a global variable, e.g. `__TEST__` which is `true` only in test runs, and skip these conditions?